tiled/src/main/kotlin/ktx/tiled/exception.kt

```diff
@@ -0,0 +1,23 @@
+package ktx.tiled
+
+import com.badlogic.gdx.maps.MapLayer
+import com.badlogic.gdx.maps.MapObject
+import com.badlogic.gdx.maps.MapProperties
+import com.badlogic.gdx.maps.objects.TextureMapObject
+import com.badlogic.gdx.maps.tiled.TiledMap
+import com.badlogic.gdx.utils.GdxRuntimeException
+
+/**
+ * [GdxRuntimeException] that is thrown when trying to access a non-existing property of a [MapProperties] instance
+ */
+class MissingPropertyException(message: String) : GdxRuntimeException(message)
+
+/**
+ * [GdxRuntimeException] that is thrown when trying to access a non-existing [MapLayer] of a [TiledMap] instance
+ */
+class MissingLayerException(message: String) : GdxRuntimeException(message)
+
+/**
+ * [GdxRuntimeException] that is thrown when trying to access a shape of a [MapObject] that do not have any shape like [TextureMapObject]
+ */
+class MissingShapeException(message: String) : GdxRuntimeException(message)
```

Comment:
Missing new line.

Reply:
Out of curiosity, what is the problem if a new line is missing? Seems like auto-formatting does not add it automatically on my side.

Reply:
[Here's a few reasons](https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file). You can configure IntelliJ to [add it automatically](https://stackoverflow.com/questions/16761227/how-to-make-intellij-idea-insert-a-new-line-at-every-end-of-file).